index.html

```diff
@@ -59,49 +41,37 @@ <h4 class="menu-title">Menu</h4>
         <ul>
           <li>
             <a href="">
-              <span class="icon icon-menu"
-                ><img src="/ProjectUI_Navbar/img/icon/my-offers.svg"
-              /></span>
+              <span class="icon icon-menu"><img src="img/icon/my-offers.svg" /></span>
               <span class="text">My Offers</span>
             </a>
           </li>
           <li>
             <a href="">
-              <span class="icon icon-menu"
-                ><img src="/ProjectUI_Navbar/img/icon/user-task.svg"
-              /></span>
+              <span class="icon icon-menu"><img src="img/icon/user-task.svg" /></span>
```

Comment:
You were using absolute paths to access assets. I've changed it to relative paths. Avoid using absolute paths. Always prefer relative paths.